package.json

```diff
@@ -37,8 +37,7 @@
   },
   "license": "MIT",
   "dependencies": {
-    "react-dev-utils": "^11.0.1",
-    "react-error-overlay": "^6.0.8",
```

Comment:
peerDeps need to be updated to include webpack 5

Reply:
done